From 90bf1743b270a72cc38dae6ef0e118f09ebe2b95 Mon Sep 17 00:00:00 2001
From: Gabriele M <moto.falcon.git@gmail.com>
Date: Sun, 1 Apr 2018 17:50:55 +0200
Subject: [PATCH 4/7] repopick: Ignore the trailing slash in change numbers

Trailing slashes cause 'patchset' to be an empty string, which
cannot be converted to an int. If we have a trailing slash, ignore
it and pick the current patchset.

Change-Id: Ifad60b7f6ff3dcfd9dd80b2e93b1f909aa360666
---
 build/tools/repopick.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/tools/repopick.py b/build/tools/repopick.py
index 248f2d79..a620f3b4 100755
--- a/build/tools/repopick.py
+++ b/build/tools/repopick.py
@@ -306,7 +306,7 @@ if __name__ == '__main__':
 
         change = int(change)
 
-        if patchset is not None:
+        if patchset:
             patchset = int(patchset)
 
         review = next((x for x in reviews if x['number'] == change), None)
-- 
2.15.1

